server/.stylelintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
unclear exactly how/when this file is read by the library. It'd be better if this could be embedded into the code to avoid spooky action at a distance

Reply:
Looking in the Stylelint documentation, there is:

> stylelint uses cosmiconfig to find and load your configuration object. Starting from the current working directory, it looks for the following possible sources:
> -  a stylelint property in package.json
>  -  a .stylelintrc file
>  -  a stylelint.config.js file exporting a JS object
>  - a stylelint.config.cjs file exporting a JS object. When running stylelint in JavaScript packages that specify "type":"module" in their package.json  


> No rules are turned on by default and there are no default values. You must explicitly configure each rule to turn it on.

> You can extend an existing configuration (whether your own or a third-party one).
> 
> Popular configurations include:
> 
>     stylelint-config-recommended - turns on just possible error rules
>     stylelint-config-standard - extends recommended one by turning on 60 stylistic rules

I used the standard configuration because I don't have much time to create a file with all the rules, and it'd be better to use the library standard config file